Source/ACE.Server/Command/Handlers/PlugInCommands.cs

```diff
@@ -0,0 +1,155 @@
+        /// </summary>
+        [CommandHandler("AceAlistLoggedOnplayers", AccessLevel.Developer, CommandHandlerFlag.RequiresWorld, 0,
+            "Sends all the players {name|True if online|AccessLevel|AccountName} connected to the server to the ACE Admin plugin.")]
+        public static void HandleListPlayers(Session session, params string[] parameters)
+        {
+            string message = "#AceAdmin.PlayerList=";
+
+            foreach (Session playerSession in WorldManager.GetAll(true))
+            {
+                message += $"{playerSession.Player.Name}:True:{playerSession.AccessLevel}:{playerSession.Account}|";
+
+            }
+
+            // remove the "|" from the end of the message.
+            message = message.Remove(message.Length - 1);
+
+            var listPlayersMessage = new GameMessageSystemChat(message, ChatMessageType.Broadcast);
+            session.Network.EnqueueSend(listPlayersMessage);
+
+        }
+
+        /// <summary>
+        /// Command to send all players in the DB. For the ACE Admin plugin
+        /// </summary>
+        [CommandHandler("AceAlistAllPlayers", AccessLevel.Developer, CommandHandlerFlag.RequiresWorld, 0,
+            "Sends ALL players {name|True if online|AccessLevel|AccountName} in the DB to the ACE Admin plugin.")]
+        public static void HandleListAllPlayers(Session session, params string[] parameters)
+        {
+            string message = "#AceAdmin.PlayerList=";
+
+            foreach (Character character in GetAllCharactersFromDB())
+            {
+                string accname = DatabaseManager.Authentication.GetAccountById(character.AccountId).AccountName;
+                uint accessLevel = DatabaseManager.Authentication.GetAccountById(character.AccountId).AccessLevel;
+                string CharName = null;
+
+                if (accessLevel > 4)
+                    CharName = "+" + character.Name;
+                else
+                    CharName = character.Name;
+
+                string onLine = "False";
+                Session playerSession = WorldManager.FindByPlayerName(CharName);
+                
+                if (playerSession != null) onLine = "True";
+
+                message += $"{CharName}:{onLine}:{accessLevel.ToString()}:{accname}|";
+                
+            }
+
+            // remove the "|" from the end of the message.
+            message = message.Remove(message.Length - 1);
+
+            var listPlayersMessage = new GameMessageSystemChat(message, ChatMessageType.Broadcast);
+            session.Network.EnqueueSend(listPlayersMessage);
+
+        }
+
+
+        public static List<Character> GetAllCharactersFromDB()
+        {
+            using (var context = new ShardDbContext())
+            {
+                var results = context.Character.
+                    AsNoTracking();
+
+                if (results != null)
+                    return results.ToList();
+
+            }
+
+            return null;
+        }
+

```

Comment:
Whats up with a lot of these weird empty lines?

Reply:
I will fix the empty lines.